### PR TITLE
added minor style changes to the  admin dashboard not complete

### PR DESCRIPTION
**Pull request by ferror18**
[x] Focused on single problem
[x] Small
[x] Message slack to ask for 2 reviewers
[] Make/update test file for each file you changed. To start just needs some semantic todolist of possible tests. (not test-driven coding yet)

##PR Title: 
[] format: [verb] [what it does] [is completed? new? fix?]

## PR Description: 
[] Outlines the motivation for the feature
[] user story link to Trello card: https://trello.com/c/J18T918C
[] Code is properly commented
[] Screenshot of completed state of feature (if applicable)

#Making Commits
[] small, atomic commit

### CODE REVIEW

File: src/components/pages/Admin/libraries.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  align-items: center;
+}
+
+.individual-library-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+/* .button-container {
```
Comment (yirano) [inline src/components/pages/Admin/libraries.css:15]: Should we bring the button-container style into the common style-component folder?

File: src/components/pages/Admin/AdminDashboard.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.admin-dashboard-container {
+  color: red;
```
Comment (Jahteo) [inline src/components/pages/Admin/AdminDashboard.css:2]: update color to branding/orange later

**Review by yirano — APPROVED** [review]

File: src/components/pages/Admin/libraries.css [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  display: flex;
+  justify-content: space-evenly;
+} */
```
Comment (Jahteo) [inline src/components/pages/Admin/libraries.css:18]: remove dead code before next pr

**Review by Jahteo — APPROVED** [review]
I like it so far, excited to see what what it looks like when it's fully fleshed.

File: src/components/pages/Admin/libraries.css (hunk 1 above)
Comment (ferror18) [reply src/components/pages/Admin/libraries.css:15]: Yes on the next pr I would be moving this CSS stylesheets and converting them into style components.

File: src/components/pages/Admin/AdminDashboard.css (hunk 2 above)
Comment (ferror18) [reply src/components/pages/Admin/AdminDashboard.css:2]: True Ill put that in the Trello card for Todo.

File: src/components/pages/Admin/libraries.css (hunk 3 above)
Comment (ferror18) [reply src/components/pages/Admin/libraries.css:18]: thanks men I missed.

**Pull request merged**